Sets the default gray background color.

This patch by Theppitak Karoonboonyanan <thep@linux.thai.net>.
This patch will not be submitted upstream.

--- a/gdraw/gxdraw.c
+++ b/gdraw/gxdraw.c
@@ -5003,7 +5003,7 @@
     GXResourceInit(gdisp,programname);
 
     gdisp->bs.double_time = GResourceFindInt( "DoubleClickTime", gdisp->bs.double_time );
-    gdisp->def_background = GResourceFindColor( "Background", COLOR_CREATE(0xf5,0xff,0xfa));
+    gdisp->def_background = GResourceFindColor( "Background", COLOR_CREATE(0xf5,0xf5,0xf5));
     gdisp->def_foreground = GResourceFindColor( "Foreground", COLOR_CREATE(0x00,0x00,0x00));
     if ( GResourceFindBool("Synchronize", false ))
 	XSynchronize(gdisp->display,true);
